_dashboards/selfhost-maps-server.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Self-host maps server
+nav_order: 6
+---
+
+# Self-host maps server
+
+The self-host maps server for OpenSearch Dashboards allows users to access the default maps service on air-gapped environments. The server provides a map manifest with map tiles and vectors URL, map tiles URL, and map vectors URL that are compatible with OpenSearch.
+
+The following steps walk through setting up and using the self-host maps server with OpenSearch Dashboards.
+
+You can access the `maps-server` image via the OpenSearch official [Docker Hub repository](https://hub.docker.com/u/opensearchproject).
+{: .note}
+
+## Pull Docker image
+
+Open your terminal and run the following command:
+
+`docker pull opensearch/opensearch-maps-server`
+
+## Set up the server
+
+You must set up the map tiles before running the server. You have two setup options: 
+
+### Option 1: Use the OpenSearch-provided maps service tiles set
+
+Create a Docker volume to hold the tiles set:
+
+`docker volume create tiles-data`
+
+Download the tiles set from the OpenSearch maps service. Planet tiles sets are available for up to 8 zoom levels `https://maps.opensearch.org/offline/planet-osm-default-z0-z8.tar.gz` and up to 10 zoom levels `https://maps.opensearch.org/offline/planet-osm-default-z0-z10.tar.gz`.
```

Comment:
Should the links be in parentheses or preceded by "at"?

Reply:
Revised as follows: Two planet tiles sets are available based on the desired zoom level:

- Zoom Level 8 (https://maps.opensearch.org/offline/planet-osm-default-z0-z8.tar.gz)
- Zoom level 10 (https://maps.opensearch.org/offline/planet-osm-default-z0-z10.tar.gz)

The planet tiles set for zoom level 10 (2 GB compressed/6.8 GB uncompressed) is approximately 10 times larger than the set for zoom level 8 (225 MB compressed/519 MB uncompressed).